src/main/java/uk/gov/justice/digital/zone/curated/CuratedZoneLoad.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.zone.curated;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import uk.gov.justice.digital.config.JobArguments;
+import uk.gov.justice.digital.exception.DataStorageException;
+import uk.gov.justice.digital.service.DataStorageService;
+import uk.gov.justice.digital.service.SourceReferenceService;
+import uk.gov.justice.digital.writer.Writer;
+import uk.gov.justice.digital.writer.curated.CuratedZoneLoadWriter;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class CuratedZoneLoad extends CuratedZone {
+
+    @Inject
+    public CuratedZoneLoad(
+            JobArguments jobArguments,
+            DataStorageService storage,
+            SourceReferenceService sourceReferenceService
+    ) {
+        this(jobArguments, sourceReferenceService, createWriter(storage));
+    }
+
+    private CuratedZoneLoad(
+            JobArguments jobArguments,
+            SourceReferenceService sourceReference,
+            Writer writer
+    ) {
+        super(jobArguments, sourceReference, writer);
+    }
+
+    private static Writer createWriter(DataStorageService storage) {
```

Comment:
If the writer classes are all annotated with `@Singleton` micronaut will take care of injecting these deps in. (See later comment).